File: Unidad3/Java/u3p8.java

package com.mycompany.Programa8_U3;
import java.util.Scanner;

public class u3p8 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese el primer numero: ");
        double numero1 = scanner.nextDouble();
        System.out.print("Ingrese el segundo numero: ");
        double numero2 = scanner.nextDouble();

        if (numero1 > numero2) {
            System.out.println("El primer numero es mayor.");
        } else if (numero2 > numero1) {
            System.out.println("El segundo numero es mayor.");
        } else {
            System.out.println("Ambos numeros son iguales.");
        }
    }
}

File: Unidad3/C#/u3p8.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Ingrese el primer número: ");
        double numero1 = double.Parse(Console.ReadLine());
        Console.Write("Ingrese el segundo número: ");
        double numero2 = double.Parse(Console.ReadLine());

        if (numero1 > numero2)
        {
            Console.WriteLine("El primer número es mayor.");
        }
        else if (numero2 > numero1)
        {
            Console.WriteLine("El segundo número es mayor.");
        }
        else
        {
            Console.WriteLine("Ambos números son iguales.");
        }
    }
}
